src/test/java/Salma_Homework22_AllSongsTests.java

```diff
@@ -0,0 +1,68 @@
+import Salma_POM.Pages.*;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class Salma_Homework22_AllSongsTests extends Salma_BaseTest{
+    @Test
+    public void shuffleSongs(){
+        LoginPage loginPage = new LoginPage(driver);
+        AllSongsPage allSongsPage = new AllSongsPage(driver);
+        //HomePage homePage = new HomePage(driver);
+        //When we declare homepage, we can use logIn method because this method returns a new
+        //instance of the home page, then we can skip logIn method in the steps.
+        HomePage homePage=loginPage.logIn();
+       //We don't need to declare basePage because all other pages extend it
+        //Step 1:Login
+        //loginPage.logIn();
+        //Step 2:click on All songs
+        homePage.clickOnAllSongs();
+        //Step 3:Shuffle
+        allSongsPage.shuffle();
+        //Step 4: Assert first song is playing
+        Assert.assertTrue(homePage.isSongPlaying());
+
+    }
+    @Test
+   public void playASongFromAllSongs(){
+        LoginPage loginPage = new LoginPage(driver);
```

Comment:
take out of from method in class  LoginPage loginPage = new LoginPage(driver);
        HomePage homePage = new HomePage(driver);
        AllSongsPage allSongsPage = new AllSongsPage(driver);